api_validation/src/main/scala/com/nvidia/spark/rapids/api/ApiValidation.scala

```diff
@@ -69,6 +69,15 @@ object ApiValidation extends Logging {
     val gpuKeys = gpuExecs.keys
     var printNewline = false
 
+    val fullClassName = ShimLoader.getSparkShims.getRapidsShuffleManagerClass
+    var shimVersion = fullClassName.split('.')(4)
```

Comment:
it would be better to just get the version from the shim itself: getSparkShimVersion